ci/scripts/run-deterministic-fuzz-test.sh

```diff
@@ -25,9 +25,57 @@ shift $((OPTIND -1))
 
 source ci/scripts/common.sh
 
+extract_error_sql() {
+  cat "$1" \
+   | grep -E "(\[EXECUTING|\[TEST)" \
+   | sed 's/.*\[EXECUTING .*\]: //' \
+   | sed 's/.*\[TEST.*\]: //' \
+   | sed 's/$/;/'
+}
+
 echo "--- Download artifacts"
 download-and-decompress-artifact risingwave_simulation .
 chmod +x ./risingwave_simulation
 
+download_and_prepare_rw "$profile" common
+
+download-and-decompress-artifact sqlsmith-reducer-"$profile" target/debug/
+mv target/debug/sqlsmith-reducer-"$profile" target/debug/sqlsmith-reducer
+chmod +x ./target/debug/sqlsmith-reducer
+
+echo "--- e2e, ci-3cn-1fe, build"
+risedev ci-start ci-3cn-1fe
```

Comment:
I don't think this is necessary. Run command should take care of strarting the cluster.